spec/win32_std_spec.cr

```diff
@@ -1,64 +1,75 @@
-# This file is autogenerated by `spec/generate_windows_spec.sh`
-# 2020-01-09 23:15:57+01:00
+# This file is autogenerated by `scripts/generate_windows_spec.sh`
+# 2020-04-20 22:36:41+02:00
 
-# require "./std/adler32_spec.cr" (failed linking)
+require "./std/adler32_spec.cr"
 require "./std/array_spec.cr"
 require "./std/atomic_spec.cr"
 require "./std/base64_spec.cr"
-# require "./std/benchmark_spec.cr" (failed codegen)
+require "./std/benchmark_spec.cr"
 # require "./std/big/big_decimal_spec.cr" (failed linking)
 # require "./std/big/big_float_spec.cr" (failed linking)
 # require "./std/big/big_int_spec.cr" (failed linking)
 # require "./std/big/big_rational_spec.cr" (failed linking)
-# require "./std/big/number_spec.cr" (failed codegen)
+# require "./std/big/number_spec.cr" (failed linking)
 require "./std/bit_array_spec.cr"
 require "./std/bool_spec.cr"
 require "./std/box_spec.cr"
-# require "./std/callstack_spec.cr" (failed codegen)
+# require "./std/callstack_spec.cr"
```

Comment:
What happened here?

Reply:
Yes, it is neither of those. It no longer fails codegen and it doesn't crash, but it just has a lot of test failures that I don't want to deal with as part of this PR. So I commented it out.

Reply:
This file has moved to call_stack_spec on master, which is already included later.